source/includes/usage-examples/write-code-examples.c

```diff
@@ -0,0 +1,142 @@
+#include <bson/bson.h>
+#include <mongoc/mongoc.h>
+#include <stdio.h>
+
+int
+main (main)
```

Comment:
```suggestion
main (void)
```

Though not included in docs, enables compiling this file.

Reply:
Thanks for catching! That typo snuck in.